apps/src/code-studio/components/header/SignInCalloutWrapper.js

```diff
@@ -24,6 +24,14 @@ export default class SignInCalloutWrapper extends React.Component {
     };
   }
 
+  componentDidMount() {
+    if (!this.state.hideCallout) {
+      this.signInElement = document.getElementById('sign_in_or_user');
```

Comment:
Great question- this was initially because the logic for onClick was here, and it made sense to explicitly remove the extra formatting in the same place we update the cookies/state/etc. But I could go either way on this!